modal/dict.py

```diff
@@ -26,9 +26,9 @@ class _Dict(_Object, type_prefix="di"):
 
     **Lifetime of a Dict and its items**
 
-    A `Dict` matches the lifetime of the app it is attached to, but invididual
-    keys expire after 30 days. Because of this, `Dict`s are best not used for
-    long-term storage. All data is deleted when the app is stopped.
+    A `Dict` matches the lifetime of the app it is attached to, but individual key-value pairs will expire 30 days after it was last added to its ​​Dict object.
```

Comment:
Nit, tense alignment

```suggestion
but an individual entry will expire 30 days after it was last added to its ​​Dict object.
```